lta/rest_server.py

```diff
@@ -0,0 +1,92 @@
+"""
+Long Term Archive REST API server.
+
+Run with `python -m lta.rest_server`.
+"""
+
+import asyncio
+import logging
+
+from tornado.ioloop import IOLoop
+
+from rest_tools.client import json_decode
+from rest_tools.server import RestServer, RestHandler, RestHandlerSetup, authenticated, catch_error
+
+from .config import from_environment
+
+
+EXPECTED_CONFIG = {
+    'LTA_REST_URL': 'localhost',
+    'LTA_REST_PORT': 8080,
+}
+
+def lta_auth(**_auth):
+    """
+    Handle RBAC authorization for LTA.
+
+    Like :py:func:`authenticated`, this requires the Authorization header
+    to be filled with a valid token.  Note that calling both decorators
+    is not necessary, as this decorator will perform authentication
+    checking as well.
+
+    Args:
+        roles (list): The roles to match
+
+    Raises:
+        :py:class:`tornado.web.HTTPError`
+    """
+    def make_wrapper(method):
```

Comment:
Missing coverage tests

Reply:
It's also not used by any code yet.  Will come with the next pull request.